Makefile

```
@@ -463,16 +470,55 @@ ${SNITCH_INSTALL_DIR}: ${TOOLCHAIN_DIR}/snitch_cluster
 
 snitch_runtime: ${SNITCH_INSTALL_DIR}
 
+${TOOLCHAIN_DIR}/spatz:
+	cd ${TOOLCHAIN_DIR} && \
+	git clone https://github.com/pulp-platform/spatz.git && \
+	cd ${TOOLCHAIN_DIR}/spatz && git checkout ${SPATZ_COMMIT_HASH} && \
+ 	git submodule update --init --recursive
+
+${SPATZ_INSTALL_DIR}: ${TOOLCHAIN_DIR}/spatz
```

checkmake (0.3.2)

[warning] Target body for "${SPATZ_INSTALL_DIR}" exceeds allowed length of 5 lines (11).

(maxbodylength)